.github/workflows/master.yml

```diff
@@ -54,10 +54,24 @@ jobs:
         run: |
           deno test -A tests/integration --config tsconfig.json --no-check=remote
 
+      - name: Run Integration Tests (remote)
+        if: ${{ matrix.os == 'ubuntu-latest'}}
+        run: |
+          docker compose -f tests/integration/docker_test/docker-compose.yml up remotes -d
+          deno test -A tests/integration --config tsconfig.json --no-check=remote -- --remoteBrowser
+          docker compose -f tests/integration/docker_test/docker-compose.yml down
+      
```

Comment:
Think it'd be better to move this to it's on job (remote specific)

eg

![image](https://user-images.githubusercontent.com/47337480/172072775-0e4da701-cf61-4f1e-9661-405b4baffdb2.png)

Reply:
note htat with the below comment, the running of the tests will take place inside the container

Reply:
I have two things on this to say, 
1. docker is not available on mac os, and also if I remember, on Windows, so we need to keep it only for Ubuntu
2. we don't want execution of tests inside the container, only the browser being there is the purpose. otherwise it won't be remote, still local.

Reply:
oh yeah thats true, keep it ubuntu then and ran outside of the container